newIDE/app/src/ExpressionAutocompletion/index.js

```diff
@@ -282,10 +286,55 @@ const getAutocompletionsForText = function(
     autocompletionTexts = getParameterChoices(
       completionDescription.getParameterMetadata()
     ).map(autocompletion => autocompletion.completion);
+  } else if (type === 'objectPointName') {
+    const objectName: string = completionDescription.getObjectName();
+    if (!objectName) {
+      return [];
+    }
+
+    const object = getObjectByName(project, scope.layout, objectName);
+    if (!object) {
+      return [];
+    }
+
+    const spriteObject = gd.asSpriteObject(object);
```

Comment:
Same in this file about checking the type

Reply:
Also add a comment before each `getType() == "Sprite"`, something like "// For now, only sprites are supporting animations." (or points).